src/elm/interval.coffee

```diff
@@ -453,3 +477,33 @@ collapseIntervals = (intervals, perWidth) ->
     collapsedIntervals.push a
     collapsedIntervals
 
+getpointSize = (interval) ->
```

Comment:
Is this function used?  As best I can tell, the one defined in the `Interval` datatype is used, but this one is not.

Reply:
It was moved to `/datatype` and should be removed here. Good catch!